templates/ui-deployment.yaml

```diff
@@ -21,6 +21,11 @@ spec:
         app: {{ template "gamebench.name" . }}
         release: "{{ .Release.Name }}"
         component: ui
+      {{- if .Values.ui.podAnnotations }}
+      annotations:
+      {{- with .Values.ui.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
I'd remove `if` statement and just use `with`.

Reply:
Fixed in https://github.com/GameBench/web-dashboard-helm-chart/pull/5/commits/d99aed12de17cd0f8419e9ca1d953d3e688d0f76 and https://github.com/GameBench/web-dashboard-helm-chart/pull/5/commits/1bd228290569172f660d30c845732229521541b2